2-build-msys-cmake.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export PKG_CONFIG_PATH="${PKG_CONFIG_PATH}:C\:/msys64/opt/mlt-6.16.0/lib/pkgconfig"
+echo "${PKG_CONFIG_PATH}"
```

Comment:
This part can be removed.

Reply:
Will remove it before it's ready to be merged.